app/components/summary_card_component.rb

```diff
@@ -20,6 +20,7 @@ def determine_rows(rows, editable)
         unless editable
           r.delete(:change_path)
           r.delete(:action_path)
+          r.delete(:action)
```

Comment:
Nice. While we're here, this method name might be improved as it implies it's determining which rows to show rather than returning a set of rows with the relevant fields knocked out or included. How about `rows_including_actions_if_editable` or sth? cc @chubberlisk

Reply:
🤔 I think that might make it a bit clearer.

Reply:
A much better name! 🚢